python/paddle/fluid/tests/unittests/test_checkpoint.py

```diff
@@ -0,0 +1,74 @@
+#   Copyright (c) 2018 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import paddle.fluid as fluid
+import unittest
+import os
+
+
+class TestCheckpoint(unittest.TestCase):
+    def setUp(self):
+        self.dirname = "/tmp/ckpt"
```

Comment:
better to use [tempfile](https://docs.python.org/2/library/tempfile.html) instead of the hard code path.